packages/dom/test/helpers.js

```diff
@@ -46,10 +38,12 @@ export function withCSSOM(rules = [], prepare = () => {}) {
   for (let rule of [].concat(rules)) {
     $style.sheet.insertRule(rule);
   }
+
+  withShadowCSSOM(rules, prepare);
 }
 
 export function withShadowCSSOM(rules = [], prepare = () => {}) {
-  let $test = getExampleShadowRoot();
+  let $test = document.getElementById('test-shadow').shadowRoot;
```

Comment:
lets keep it consistent, have it inside `withCSSOM` and pass `options` which contains `shadow` as one of the option.